PusherServer/NotifyResult.cs

```diff
@@ -0,0 +1,45 @@
+﻿using PusherServer.Exceptions;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Web.Script.Serialization;
+
+namespace PusherServer
+{
+    internal class NotifyResult : RequestResult, INotifyResult
+    {
+        private readonly int _numberOfSubscribers;
+
+        /// <summary>
+        /// Constructs a new instance of a NotifyResult based upon a passed in Rest Response
+        /// </summary>
+        /// <param name="response">The Rest Response which will form the basis of this Notify Result</param>
+        public NotifyResult(IRestResponse response) : base(response)
+        {
+            NotifySubscriberData subscriberData = null;
+            try
+            {
+                JavaScriptSerializer serializer = new JavaScriptSerializer();
+                subscriberData = serializer.Deserialize<NotifySubscriberData>(response.Content);
+            }
+            catch (Exception)
```

Comment:
This is catching any exception, so your error message may be incorrect.  Do we need to catch and throw our own exception here, or would the one that's been thrown be better?